customer-wallet/src/containers/Customer/Transactions/TransactionHistory/index.js

```diff
@@ -24,11 +27,47 @@ class TransactionHistory extends Component {
     location: PropTypes.shape({
       pathname: PropTypes.string.isRequired
     }).isRequired,
-    getRates: PropTypes.func.isRequired
+    getRates: PropTypes.func.isRequired,
+    address: PropTypes.string,
+    getTransactionHistory: PropTypes.func.isRequired
   }
 
+  unconfirmed = new WebSocket('wss://socket.blockcypher.com/v1/btc/main')
+  confirmed = new WebSocket('wss://socket.blockcypher.com/v1/btc/main')
+
   componentDidMount () {
+    const { address } = this.props
+
     this.getFxRates()
+
+    // confirmed
+    this.confirmed.onmessage = (event) => {
+      const data = JSON.parse(get(event, 'data', '{}'))
+      getTransactionHistory(address)
```

Comment:
This will result in all of the transaction history being fetched again on any event received on the websocket. We should not do this. Instead we should append the new transaction received and let react's state management result in new rendering of the transaction history.

Reply:
I understand this problem, and it is just temporary solution. I will change it today